<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>19</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter Server Listening Port</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>22</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lEportno">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>44556</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_server_start">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter Remote PC Details</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LeRemoteIP">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LeRemotePort">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>44557</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_connect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>319</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Result | Status</string>
    </property>
    <widget class="QLabel" name="label_Result">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Game Not Started</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_Game"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onExitMenuClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Game</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnNewGameClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onExitMenuClicked()</slot>
  <slot>OnNewGameClicked()</slot>
 </slots>
</ui>
